bin/log-conventions.js

```diff
@@ -0,0 +1,57 @@
+require('colors')
```

Comment:
Nice :) Output looks really good in the ci! Failing tests are now visually appealing ^^